src/components/DocumentationTopic/DocumentationNav/LanguageToggle.vue

```diff
@@ -125,12 +125,12 @@ export default {
   },
   mounted() {
     // on resize, re-calculate the width of the select.
-    const cb = debounce(async () => {
+    const cb = throttle(async () => {
```

Comment:
Could you explain me this change? Thanks

Reply:
I think Bea noticed the language toggle was not getting the new screen size, when we resize the page by dragging the adjustable sidebar. Debounce can skip the last function hit, while throttle will always fire the last function invocation.